cmd/generate/config/rules/atlassian.go

```diff
@@ -11,15 +11,26 @@ func Atlassian() *config.Rule {
 	r := config.Rule{
 		Description: "Detected an Atlassian API token, posing a threat to project management and collaboration tool security and data confidentiality.",
 		RuleID:      "atlassian-api-token",
-		Regex: utils.GenerateSemiGenericRegex([]string{
-			"atlassian", "confluence", "jira"}, utils.AlphaNumeric("24"), true),
+		Regex: utils.MergeRegexps(
+			utils.GenerateSemiGenericRegex(
+				[]string{"atlassian", "confluence", "jira"},
+				utils.AlphaNumeric("24"),
+				true,
+			),
+			utils.GenerateSemiGenericRegex(
+				[]string{"atlassian", "confluence", "jira"},
+				`[A-Za-z0-9_\-=]{192}`,
+				false,
+			),
+		),
```

Comment:
Might as well add entropy while we're changing this. Every old token I've found is _exactly_ 4.00, newer ones are > 5. I think 3.5 is a good option.
```suggestion
		),
		Entropy: 3.5,
```

Reply:
Added. Thank you!